### PR TITLE
Improves SEC-460: update the updater

### PR DESCRIPTION
**Pull request by Screenfeed**
Enables beta updates.
To merge AFTER the SEC-331 branch.

### CODE REVIEW

File: inc/admin/ajax-post-callbacks.php [hunk 1]
```diff
@@ -741,36 +761,109 @@ function secupress_global_settings_api_request_succeeded( $response ) {
...
+			__( 'Your license key seems invalid. You may want %s.', 'secupress' ),
+			sprintf( $account_link, __( 'to verify these infos', 'secupress-pro' ) )
+		),
+		'request_error'       => __( 'Something on your website is preventing the request to be sent.', 'secupress' ),
+		/** Translators: %s is a "our support team" link. */
+		'server_error'        => sprintf( __( 'Our server is not accessible at the moment, please try again later or contact %s.', 'secupress' ), $support_link ),
+		/** Translators: %s is a "our support team" link. */
+		'server_bad_response' => sprintf( __( 'Our server returned an unexpected response and might be in error, please try again later or contact %s.', 'secupress' ), $support_link ),
+		/** Translators: %s is a "our support team" link. */
+		'invalid_api_request' => sprintf( __( 'There is a problem with your license key, please contact our support team.', 'secupress' ), $support_link ),
```
Comment (JulioPotier) [inline inc/admin/ajax-post-callbacks.php:816]: missing %s

File: inc/admin/ajax-post-callbacks.php [hunk 2]
```diff
@@ -741,36 +761,109 @@ function secupress_global_settings_api_request_succeeded( $response ) {
...
+	}
 
-			add_settings_error( 'general', 'response_error', __( 'You\'ve used as many as your license allows, you may want to upgrade your license to add more sites.', 'secupress' ) );
+	$all_errors = array_merge( $api_errors, $edd_errors );
 
-		} else {
-			add_settings_error( 'general', 'response_error', __( 'Something may be wrong with your license, please take a look at your account or contact our support team.', 'secupress' ) );
-		}
+	// Generic message.
+	$all_errors['license_error'] = __( 'Something may be wrong with your license, please take a look at your account or contact our support team.', 'secupress' );
```
Comment (JulioPotier) [inline inc/admin/ajax-post-callbacks.php]: missing links ?

Comment (Screenfeed) [reply inc/admin/ajax-post-callbacks.php]: Indeed, I will add them.

File: inc/admin/ajax-post-callbacks.php (hunk 1 above)
Comment (Screenfeed) [reply inc/admin/ajax-post-callbacks.php:816]: Good catch!

**Pull request merged**